requirements.txt

```diff
@@ -1,4 +1,4 @@
-pyfrc>=2018.2.3
+pyfrc>=2018.0.0,<2019.0.0
 robotpy-ctre>=2018.3.1
```

Comment:
Can you do the same thing for all the other robotpy packages with year versions?